### DISCUSSION TITLE
Plugin de cuenta regresiva

### DISCUSSION THREAD

**Discussion by giscus** [zkreations/data#26] [Announcements]
# Plugin de cuenta regresiva

Reloj de cuenta regresiva, widget de cuenta regresiva para eventos especiales o indicar lo que falta para culminar el año.

https://www.zkreations.com/2017/10/widget-cuenta-atras-blogger.html

**Comment by TakoMics** [comment]
Dios SANTO, ¡Esto es increíble, Daniel! Widgets que se instalan desde un botón. Definitivamente este es otro nivel.

Por otro lado, ¿esta sección traerá los anteriores Widgets de versiones pasadas? Como el Widget de YouTube con Lazy Load, las categorías desplegables, cajas de suscripción (o botones de suscripción) para MailChimp, imágenes en carruseles, etc?

**Reply by danieIabel** [reply]
Claro amigo, mi objetivo es poder traer todos los widgets que sean posibles con este método, es mucho mas limpio y su desinstalación no deja residuos que es lo que busco.

**Comment by gianles** [comment]
genial aporte, aun recuerdo la versión que creaste hace ya un buen tiempo..

**Reply by danieIabel** [reply]
Estoy tratando de trasladar esos widgets a este formato en donde solo se instala en un clic. Se que no se podrá con todos ya que esta limitado a un solo tipo widget, pero espero encontrar alguna solución. Gracias por comentar amigo!

**Comment by danieIabel** [comment]
He actualizado la lógica de instalación del widget, ahora podrás instalar tantos contadores como quieras y el código javascript solo se ejecutará una vez.

**Comment by jesusmames** [comment]
Lo primero, muchas gracias.
Y una pregunta, ¿es posible colocar esta cuenta regresiva en una única entrada o página del blog?

**Reply by danieIabel** [reply]
Buenos dias amigo. Claro que puede insertarlo en una entrada o en cualquier otra parte que no sea un widget. Para ello debe [ir al ejemplo en codepen](https://codepen.io/zkreations/pen/QWYOBeo) y copiar el código. En el caso de CSS debe asegurase de copiar el código compilado:

![image](https://github.com/zkreations/data/assets/40010778/08a7ff23-cf9a-4103-b9cf-d93dc1c8467b)

**Reply by jesusmames** [reply]
Vale. 
Muchas gracias.
